app/Http/Resources/RequestNcpoDetailedResource.php

```
@@ -0,0 +1,27 @@
+     * @return array<string, mixed>
+     */
+    public function toArray(Request $request): array
+    {
+        return [
+            'id' => $this->id,
+            'date' => $this->date,
+            'ncpo_no' => $this->ncpo_no,
+            'po_id' => $this->po_id,
+            'justification' => $this->justification,
+            'created_by' => $this->created_by,
+            'items' => RequestNcpoItemResource::collection($this->items),
```

PHPMD (2.15.0)

Avoid using static access to class 'App\Http\Resources\RequestNcpoItemResource' in method 'toArray'. (undefined)

(StaticAccess)